src/man/sssd.conf.5.xml

```diff
@@ -4443,10 +4443,11 @@ ldap_user_extra_attrs = phone:telephoneNumber
                                 <quote>ldap</quote>, <quote>AD</quote> or
                                 <quote>ipa</quote>.</para>
                             </listitem>
-                            <listitem condition="with_files_provider">
-                                <para>The RULE_NAME for the <quote>files</quote>
-                                provider which tries to find a user with the
-                                same name.</para>
+                            <listitem>
+                                <para>If maprule is not set and provider is
+                                <quote>proxy</quote> or <quote>files</quote>, the
```

Comment:
I think we need two versions of this '<para>':
1) with_files_provider => as you wrote
2) else => exclude "or <quote>files</quote>"

Reply:
I mean, when SSSD is built without files provider, we shouldn't mention "provider is files" in the man page. It is confusing.

Reply:
Hi,

I think you can use[ `<phrase>` ](https://tdg.docbook.org/tdg/4.5/phrase) to make "or \<quote\>files\</quote\>" conditional.

HTH

bye,
Sumit

Reply:
Hi,

I guess every white-space following an opening tag is skipped, i.e. every white-space after `<phrase>`. But a white-space between tags is preserved, so `</quote> <phrase ...` would work.

However, I think using `&nbsp;` is a good idea here to indicate that the first character of this phrase should be a white-space. The only drawback I see is that we would not allow a line break between "proxy" and "or", but given the use-case I think this is acceptable.

bye,
Sumit